packages/babel-plugin-proposal-do-expressions/test/fixtures/do-expression-switch-case/fallthrough-multiblock/exec.js

```diff
@@ -0,0 +1,34 @@
+const x = (n) => do {
+  switch (n) {
+    case 0: 
+      { "a"; }
+      { "b"; };
+    case 1: 
+      { "c"; }
+      break;
+      { "d"; };
+    case 2: 
+      "a";
+      "b";
+    case 3:
+      {}
+      { break; }
```

Comment:
@nicolo-ribaudo i need some help over here, i've figured this as a plausible test case, but I not sure how i should transform this.

Previously, what I did is to find the `BreakStatement`, and turn the statement before `BreakStatement` as a completionPath, which would then be transformed into a return statement.

Problem I've created right now is that the previous statement could be anything from `BlockStatement` to nested `BlockStatement`s, and I need to find the `"last statement"` which I am kind stuck on how I should go for this...
not entirely sure I am in the right direction, that's why I am seeking some help over here

Reply:
I'm not sure about how to solve it, but it isn't a problem just for switchs. It's a problem for everything which uses `break`.

e.g.
```js
let a = do {
  while (1) {
    1;
    break;
    2;
  }
};
```

I think that we should try to fix it separately since it is a really hard problem and this PR is already a big improvement.

Reply:
OK I guess I shall disable that test case then